input.go

```diff
@@ -75,6 +75,11 @@ type VMInput struct {
 type ContractCreateInput struct {
 	VMInput
 
+	// NewContractAddress is the address of the new contract to be created.
+	// An empty NewContractAddress will cause the hook to be called.
+	// If the hook also returns an empty address, the VM will decide the new address.
+	NewContractAddress []byte
```

Comment:
I think we can remove this. We should rely only on the hook.

Reply:
done